demo/resources/view.html

```diff
@@ -1,6 +1,23 @@
-<!-- ko-viewmodel: import default from './viewmodel' -->
-<!-- ko-bindinghandler: import koko from './bindinghandler' -->
-
-<div data-bind="koko: [prop]">
+<!-- ko-viewmodel: import Vymodell from './viewmodel' -->
+<!-- ko-bindinghandler: import koko1 from './bindinghandler' -->
+<!-- ko-bindinghandler: import * as koko2 from './bindinghandler' -->
+<!-- ko-bindinghandler: import typeof koko3 from './bindinghandler' -->
+<!-- ko-bindinghandler: import typeof * as koko4 from './bindinghandler' -->
+<!-- ko-bindinghandler: import { koko5 } from './bindinghandler' -->
+<!-- ko-bindinghandler: import { koko as KOKO6 } from './bindinghandler' -->
+<!-- ko-bindinghandler: import { typeof koko7 } from './bindinghandler' -->
+<!-- ko-bindinghandler: import { typeof koko as KOKO8 } from './bindinghandler' -->
```

Comment:
Too many binding handler imports.

Reply:
This is to try out/document all import types.

Reply:
It should be tested in spec. I'll allow it because of #110.